docs/aspects/teambit.react/react/react.docs.md

```diff
@@ -132,23 +137,21 @@ export class CustomReactExtension {
 }
 ```
 
-> To override any specific configuration it's recommended to create a config file for the specific tool and import it to any of the **transformers**.
+> To override any specific configuration it's recommended to create a separate config file with your customized configuration and import it into the main `.extension.ts` file where you can supply it to the relevant **transformers**. See the tsconfig example in the above snippet.
 
 ### Composing tools and services
 
-Environments use Environment Services by implementing a special class of methods called Service Handlers.  
-You can override an environment's compiler replacing its Compiler Service Handler with the method `getCompiler()`.  
+Environments use Environment Services by implementing a special class of methods called Service Handlers.
+The supplied transformers allow overriding of these services - for instance the `overrideCompiler` transformer allows overriding the environment's `getCompiler()` method.
 [Find all service handlers here](#service-providers-api-docs).
```

Comment:
if you commented the APIs, this link will be broken. so perhaps remove it?

Reply:
Right, removing.